index.bs

```diff
@@ -492,22 +504,30 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
 1. Set |matchingSources| to the result of [=list/sort in descending order|sorting=] |matchingSources|
     in descending order, with |a| being less than |b| if any of the following are true:
       * |a|'s [=attribution source/priority=] is less than |b|'s [=attribution source/priority=].
-      * |a|'s [=attribution source/priority=] is equal to |b|'s [=attribution source/priority=] and |a|'s 
+      * |a|'s [=attribution source/priority=] is equal to |b|'s [=attribution source/priority=] and |a|'s
          [=attribution source/source time=] is less than |b|'s [=attribution source/source time=].
 1. Let |sourceToAttribute| be the first item in |matchingSources|.
 1. If |trigger|'s [=attribution trigger/dedup key=] is not null and |sourceToAttribute|'s
     [=attribution source/dedup keys=] [=list/contains=] it, return.
+1. Let |report| be the result of running [=obtain a report=] with |sourceToAttribute| and |trigger|.
+1. If |sourceToAttribute|'s [=attribution source/number of reports=] value is greater than or equal to the
+    user agent's [=max reports per source=] value, then:
+    1. Let |matchingReports| be all entries in the [=attribution report cache=] where all of the following are true:
+         * entry's [=attribution report/report time=] and |report|'s [=attribution report/report time=] are equal.
+         * entry's [=attribution report/source=] and |report|'s [=attribution report/source=] are the same.
+    1. Set |matchingReports| to the result of [=list/sort in ascending order|sorting=] |matchingReports|
+        in ascending order, with |a| being less than |b| if |a|'s [=attribution report/trigger priority=] is less than |b|'s [=attribution report/trigger priority=].
+    1. Let |lowestPriorityReport| be the first item in |matchingReports|.
+    1. If |report|'s [=attribution report/trigger priority=] is less than or equal to |lowestPriorityReport|'s [=attribution report/trigger priority=], return.
```

Comment:
I think we still want to delete the rest of sources in matching sources in this case. The browser properly ran the attribution logic, it just chose not to send the report.

Reply:
Is that different from what's currently implemented?

https://source.chromium.org/chromium/chromium/src/+/master:content/browser/conversions/conversion_storage_sql.cc;l=521-526;drc=09b4249fe83da3613c09b66cdab1d44dd40d21fd

Reply:
Yes it is. Looking further, I think we are actually guaranteed matchingSource only contains sourceToAttribute at this point:

 - other sources would have gotten deleted on the first trigger
 - new sources afterwards would have deleted sourceToAttribute due to the store an attribution source logic

Does that seem right to you?

In any case think we can leave this as is.  Might be worth adding a note here to explain that.

Reply:
Yes, that seems right. What's the proper way to add a note like that?

Reply:
I'm not really sure. One concern is that it could get out of date if algorithms change. I think what you have is fine.

Reply:
Ack.